src/main/java/eu/execom/pomodoroTeam/controllers/LoginController.java

```diff
@@ -34,22 +45,50 @@ public Principal user(Principal principal) {
         return principal;
     }
 
-    @GetMapping("/user/getAll")
-    public Iterable<UserEntity> getAllUsers() {
-        return userRepository.findAll();
+    @GetMapping("/getAll")
 
+    public ResponseEntity<List<UserEntity>> getAllUsers() {

```

Comment:
Please avoid having floating annotations, there should not be any spacing between annotations and classes, fields or methods to which they are related.